_source/_posts/2018-11-15-node-express-typescript.md

```diff
@@ -443,21 +443,9 @@ You will use the `.env` for much more configuration information as the project g
 
 ## Easily Add Authentication to Node and Express
 
-Adding user registration and login (authentication) to any application is not a trivial task. The good news is Okta makes this step very easy. To begin, create a free developer account with Okta. First, navigate to [developer.okta.com](https://developer.okta.com/) and click the **Create Free Account** button, or click the [Sign Up](https://developer.okta.com/signup/) button.
+Adding user registration and login (authentication) to any application is not a trivial task. The good news is Okta makes this step very easy. 
 
-{% img blog/node-express-typescript/add-application-00.jpg alt:"Sign up for free account" width:"800" %}{: .center-image }
-
-After creating your account, click the **Applications** link at the top, and then click **Add Application**.
-
-{% img blog/node-express-typescript/add-application-01.jpg alt:"Create application" width:"500" %}{: .center-image }
-
-Next, choose a **Web Application** and click **Next**.
-
-{% img blog/node-express-typescript/add-application-02.jpg alt:"Create a web application" width:"800" %}{: .center-image }
-
-Enter a name for your application, such as *Guitar Inventory*. Verify the port number is the same as configured for your local web application. Then, click **Done** to finish creating the application.
-
-{% img blog/node-express-typescript/add-application-03.jpg alt:"Application settings" width:"800" %}{: .center-image }
+{% include setup/cli.md type="web" loginRedirectUri="http://localhost:8080/authorization-code/callback" %}
```

Comment:
This isn't rendering the redirect URL correctly. I'm looking into how to fix now.